How-GitHub-Issue-Moderators-AKA-Issue-Mods-Work.md

```diff
@@ -43,7 +43,7 @@ PRs must meet the following requirements:
 8. Code should pass all tests and linting
 9. Code should be of general high quality and a needed change or improvement
 
-If the issue does not meet one or more of these requirements, note the deficiencies in a comment and/or highlight lines.  New Contributors may be referred to the [HelpContributors](https://gitter.im/FreeCodeCamp/HelpContributors) Chat room.  At at the Mod's discretion the issue may be closed.
+If the issue does not meet one or more of these requirements, note the deficiencies in a comment and/or highlight lines. New Contributors may be referred to the [HelpContributors](https://gitter.im/FreeCodeCamp/HelpContributors) Chat room. The PR may alternatively be closed at a Mod's discretion.

```

Comment:
Sorry, I mean to say that it may ALSO be closed.  Gah.  What is English so hard?

How about:
> If the issue does not meet one or more of these requirements, note the deficiencies in a comment and/or highlight lines. New Contributors may be referred to the [HelpContributors](https://gitter.im/FreeCodeCamp/HelpContributors) Chat room. If the issues are too large to be easily fixed (wrong branch, merge commits, etc) the PR may also be closed, at the Mod's discretion.